src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const f = (x) => {
+    const g = (y) => {
```

Comment:
not the best names actually)
```suggestion
  const setFirstNumber = (x) => {
    const addSecondNumber = (y) => {
```